app/api/server.go

```diff
@@ -146,7 +148,14 @@ func (s *Server) getFeedCtrl(w http.ResponseWriter, r *http.Request) {
 
 	// replace link to UI page
 	if s.Conf.System.BaseURL != "" {
-		rss.Link = s.Conf.System.BaseURL + "/feed/" + feedName
+		baseURL := s.Conf.System.BaseURL
+		rss.Link = baseURL + "/feed/" + feedName
+		rss.ItunesImage = feed.ItunesImg{
+			URL: baseURL + "/image/" + feedName,
```

Comment:
we shouldn't form this image URL automatically but rather conditionally from s.Conf.Feeds[feedName].Image (if not empty)

Reply:
got it, fixed

Reply:
almost ;) the image url should be `baseURL+s.Conf.Feeds[feedName].Image` instead of generic `baseURL + "/image/" + feedName`